src/stdlib_experimental_error.f90

```diff
@@ -22,7 +22,7 @@ end subroutine error_stop
 contains
 
 subroutine check(condition, msg, code, warn)
-    !! Checks the value of a logical condition. ([spec](../page/specs/stdlib_experimental_error.html#description))
```

Comment:
@jvdp1 By the way, this *is* redundant, but unless you add the `!! summary: ...` meta data to the docstrings explicitly then the first paragraph is used for summary text on pages [like this one](https://stdlib.fortran-lang.org/lists/modules.html). (Look at the "description" column.) So I added an abbreviated link to the spec in the summary/first paragraph so that you could easily navigate to the spec from the [overview page](https://stdlib.fortran-lang.org/lists/modules.html).

I'm happy either way, but I thought it was convenient to have there.

Reply:
Thanks @zbeekman for the review. I agree with your point-of-view, and also prefer that the link to the spec appears in the "description" column.  So I modified all the files such that the link to the spec appears in the description column for all modules and procedures (when available).

Reply:
I haven't looked at the generated documentation, but your latest commit looks like it will do what you intend it to! Thanks again for working on this!